src/engagement_db_to_analysis/configuration.py

```diff
@@ -0,0 +1,13 @@
+from dataclasses import dataclass
+
+
+@dataclass()
+class DatasetTypes:
+    DEMOGRAPHIC = "demographic"
+    RESEARCH_QUESTION_ANSWER = "research_question_answer"
+
+
+@dataclass
+class AnalysisConfiguration:
+    engagement_db_dataset: str
```

Comment:
(If this is a str[] we get a lot more flexibility e.g. we can combine locations (as was needed by gpsdd)

Reply:
Quite right. Updated to list of strings.